Makefile

```diff
@@ -439,7 +439,7 @@ microk8s: check-microk8s
 ci-precheck: precheck
 	$(QUIET) $(MAKE) $(SUBMAKEOPTS) -C bpf build_all
 
-precheck: ineffassign logging-subsys-field
+precheck: logging-subsys-field
```

Comment:
just a note on this, I make a decision to skip running _lint target_ here, as I was facing issue running it in Jenkins precheck (Jenkins prefix -> docker-compose.precheck -> make ci-precheck -> make precheck), which might require to install golangci-lint in _cilium-builder_ docker image.

As the same check is done github action now, and developer can easily run _make lint_ in vagrant box, we are still having some guarantee on codebase. We can enable this as part of _pre-check_ in subsequent PR.